sp800-63b/sec5_authenticators.md

```diff
@@ -317,15 +317,15 @@ CSPs SHALL provide subscriber instructions on how to appropriately  protect the
 
 #### <a name="throttle"></a>5.2.2. Rate Limiting (Throttling)
 
-When required in the authenticator type descriptions under Section 5.1, the verifier SHALL implement controls to protect against online guessing attacks. Unless otherwise specified in the description of a given authenticator, the verifier SHALL effectively limit online attackers to no more than 100 consecutive failed attempts on a single account.
+When required in the authenticator type descriptions under [Section 5.1](#reqauthtype), the verifier SHALL implement controls to protect against online guessing attacks. Unless otherwise specified in the description of a given authenticator, the verifier SHALL effectively limit online attackers to no more than 100 consecutive failed attempts on a single account.
 
 Additional techniques MAY be used to prioritize authentication attempts that are likely to come from the subscriber over those that are more likely to come from an attacker:
 
-- Requiring the claimant to complete a Completely Automated Public Turing test to tell Computers and Humans Apart (CAPTCHA) before attempting authentication
+- Requiring the claimant to complete a Completely Automated Public Turing test to tell CAPTCHA before attempting authentication.
```

Comment:
New wording doesn't make sense. The old wording provided the meaning of the abbreviation CAPTCHA, and the new one doesn't. Capitalization of the words was based on those included in the abbreviation.